src/storage_network/types.rs

```diff
@@ -344,6 +368,21 @@ impl std::fmt::Debug for NetworkCommand {
                 .debug_struct("SetMetrics")
                 .field("metrics", &"<metrics>")
                 .finish(),
+            Self::RegisterRaftHandler { .. } => f
```

Comment:
We're leaking parts of raft into the network layer, this will make it difficult if we decide to use this network layer with something other than raft in the future.